StepperMotor/StepperMotor.c

```diff
@@ -74,13 +74,16 @@ void StepperMotor_StepForward(void)
 	
 	if(StepperMotor_GlobalPosition == SYRINGE_LENGTH)
 	{
```

Comment:
LEDClear is only used for these 2 scenarios.  It shouldn't be called if the stepper motor is going to continue to spin.  I can put it back in the main loop, but then I will also move the LEDClear from the StepperMotor_StepBackward into the main as well to keep consistency.